src/plotconfig.jl

```diff
@@ -253,7 +254,7 @@ function PlotConfig(T::Val{:erpimage})
         cfg;
         layout = (; use_colorbar = true),
         colorbar = (; label = "Voltage [µV]"),
-        axis = (xlabel = "Time [s]", ylabel = "Trials"),
+        axis = (xlabel = "Time samples", ylabel = "Trials"),
```

Comment:
this is not correct - if you call plot_erpimage with a dataframe, you should get time[s] if you call it with a matrix, you should get samples. Therefore, yo have to update this in the conversion function

Reply:
yeah, but  we currently do not accept DataFrames in this function and as is understood it is not really necessary

Reply:
ah sorry - but you can call it with `times, data` - maybe then switch to `Time` without unit, because we dont know the unit?

Reply:
okay i will delete all units

Reply:
why? in some cases we know the units, no? in other plots you also assume units, so to some degree I think this is fine?

Reply:
I think time units are not necessary (could be misleading), nut about mV i am not sure